lib/api/api.dart

```diff
@@ -6,6 +6,8 @@ import 'package:clnapp/model/app_model/pay_invoice.dart';
 
 import 'package:clnapp/model/app_model/list_send_pays.dart';
 
+import '../model/app_model/generate_invoice.dart';
```

Comment:
```suggestion
import 'package:clnapp/model/app_model/generate_invoice.dart';
```

always absolute path